core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/catalog/domain/ProductImpl.java

```diff
@@ -264,12 +264,12 @@ public class ProductImpl implements Product, ProductAdminPresentation, Status, A
             tooltip = "ProductImpl_Canonical_Url_Tooltip")
     protected String canonicalUrl;
 
-    @Column(name = "USE_DEFAULT_SKU_IN_INVENTORY")
-    @AdminPresentation(friendlyName = "ProductImpl_useDefaultSkuInInventory",
-            group = GroupName.Miscellaneous, order = 7000,
-            tooltip = "ProductImpl_useDefaultSkuInInventory_Tooltip",
-            defaultValue = "false")
-    protected Boolean useDefaultSkuInInventory = false;
+//    @Column(name = "USE_DEFAULT_SKU_IN_INVENTORY")
```

Comment:
probably worth adding TODO that in 6.3 this should be uncommented and Weave class should be deleted

Reply:
Is it worth going ahead and creating a 6.3 branch and fixing it now?

Reply:
https://github.com/BroadleafCommerce/QA/issues/4686